smslibrary/src/main/java/com/eis/smslibrary/SMSSentBroadcastReceiver.java

```diff
@@ -0,0 +1,78 @@
+package com.eis.smslibrary;
+
+import android.app.Activity;
+import android.app.PendingIntent;
+import android.content.BroadcastReceiver;
+import android.content.Context;
+import android.content.Intent;
+import android.telephony.SmsManager;
+
+import androidx.annotation.NonNull;
+
+import com.eis.smslibrary.listeners.SMSSentListener;
+
+/**
+ * Broadcast receiver for sent messages, called by Android Library.
+ * Must be instantiated and set as receiver with context.registerReceiver(...)
+ */
+class SMSSentBroadcastReceiver extends BroadcastReceiver {
+
+    private SMSSentListener listener;
+    private SMSMessage message;
+
+    /**
+     * Constructor for the custom {@link BroadcastReceiver}.
+     *
+     * @param message  message that will be sent.
+     * @param listener listener to be called when the operation is completed successfully or not.
+     */
+    SMSSentBroadcastReceiver(@NonNull final SMSMessage message,@NonNull final SMSSentListener listener) {
+        this.listener = listener;
+        this.message = message;
+    }
+
+    /**
+     * @param listener called when the message is sent.
+     */
+    void setListener(@NonNull final SMSSentListener listener) {
+        this.listener = listener;
+    }
+
+    /**
+     * @param message passed to the listener when it is sent.
+     */
+    void setMessage(@NonNull final SMSMessage message) {
+        this.message = message;
+    }
+
+    /**
+     * This method is subscribed to the intent of a message sent, and will be called whenever a message is sent using this library.
+     */
```

Comment:
Here there isn't the description of params..did you forget it or is it ok like this because it's an @Override?

Reply:
I have no idea of what the params contain because it's an override, so I only wrote the description of the method.